examples/personas-bank/README.md

```diff
@@ -73,20 +70,14 @@ With the configuration created, run `terraform init` and `terraform apply` to pr
 An SOCKS5 proxy over an SSH channel on your workstation can be used
 to access the TFE deployment from outside of the AWS network. The
 following example demonstrates how to establish a SOCKS5 proxy using
-Bash, a bastion host virtual machine, and an Internet browser.
+Bash, the AWS CLI, and an Internet browser.
 
 First, establish the SOCKS5 proxy. The following command creates a
 proxy listening to port 5000 and bound to localhost which forwards
 traffic through one of the compute instances in the TFE delpoyment.
 Be sure to change the values in between `< >`:
 
 ```bash
```

Comment:
Did you mean for this to be an empty block?

Reply:
Nope! I forgot that I need to finish those instructions 😅